result/yoda.js

```diff
@@ -0,0 +1,74 @@
+if ("red" === color) {
+    // ...
+}
+if (color === "red") {
+    // ...
+}
+/*eslint yoda: "error"*/
+
+if ("red" === color) {
+    // ...
+}
+
+if (true == flag) {
+    // ...
+}
+
+if (5 > count) {
+    // ...
+}
+
+if (-1 < str.indexOf(substr)) {
```

Comment:
__[development][eslint]__
* Error - 'str' is not defined. (no-undef)
* Error - 'substr' is not defined. (no-undef)